stl/CMakeLists.txt

```diff
@@ -552,20 +552,38 @@ function(add_stl_dlls D_SUFFIX THIS_CONFIG_DEFINITIONS THIS_CONFIG_COMPILE_OPTIO
     add_dependencies(msvcp${D_SUFFIX}_implib msvcp${D_SUFFIX} msvcp_1${D_SUFFIX} msvcp_2${D_SUFFIX} msvcp${D_SUFFIX}_atomic_wait msvcp${D_SUFFIX}_codecvt_ids)
     set_target_properties(msvcp${D_SUFFIX}_implib PROPERTIES STATIC_LIBRARY_OPTIONS "/NOLOGO;/NODEFAULTLIB;/IGNORE:4006;$<TARGET_LINKER_FILE:msvcp${D_SUFFIX}>;$<TARGET_LINKER_FILE:msvcp_1${D_SUFFIX}>;$<TARGET_LINKER_FILE:msvcp_2${D_SUFFIX}>;$<TARGET_LINKER_FILE:msvcp${D_SUFFIX}_codecvt_ids>;$<TARGET_LINKER_FILE:msvcp${D_SUFFIX}_atomic_wait>")
     set_target_properties(msvcp${D_SUFFIX}_implib PROPERTIES ARCHIVE_OUTPUT_NAME "msvcprt${D_SUFFIX}")
+
+    # HACK: for benchmarking, we need to export the standard library since we link google benchmark to the stl targets,
+    # and CMake requires all targets depended upon by exported targets to themselves be exported.
+    set(msvcp${D_SUFFIX}_all_targets
+        boost_math
+        stl_alias_objects
+        msvcp${D_SUFFIX}_implib
+        CACHE INTERNAL ""
+    )

```

Comment:
again, we could just use the usual runtime library settings if we set up LIB and INCLUDE correctly (I suppose this is a bit cursed)